bin/varnishtest/tests/r03241.vtc

```diff
@@ -43,11 +47,10 @@ logexpect l1 -v v1 -g raw {
 
 client c1 {
 	txreq -hdr "Host: foo"
-	rxresphdrs
+	rxresp
+	# XXX this is actually wrong (missed include)
+	expect resp.bodylen == 57
```

Comment:
This comment should be changed:

> 15:24 < phk> so this comment should really say "if you want this to be correct, set esi_include_onerror ?

It will be done in a separate commit to keep the revert minimal.